content/2.0/index.html

```diff
@@ -0,0 +1,117 @@
+---
+:layout: post
+:title: Jenkins 2.0
+---
+
+
+<p>
+  Jenkins 2.0 is an upcoming major new version of Jenkins.
+</p>
+
+
+<h2>Highlights</h2>
+<!-- TODO: learn how to reuse site building blocks so that these look nicer -->
+<style>
+  #highlights DT {
+    font-weight: bold;
+    margin-top: 1em;
+  }
+  #highlights DD {
+    padding-left: 1em;
+    margin-bottom: 1em;
+  }
+</style>
+<dl id=highlights>
+  <!-- TODO: these should be partials -->
+  <dt>Pipeline as Code keeps your configuration <a href="https://en.wikipedia.org/wiki/Don%27t_repeat_yourself">DRY</a>
+  <dd>
+    New pipeline functionality in Jenkins allows you to define your job configuration
+    in a textual format, version controlled along side your source code. It lets
+    you define delivery pipelines with ease, copes well when pipelines get complex,
+    and far less repetitive when you have lots of repositories.
+
+  <dt>Out of the box experience: You are in good hands
+  <dd>
+    Jenkins 2.0 starts off with much more modern and richer set of functionalities that are catered
+    to today's development best practices. These functionalities are carefully chosen from the vast
+    pool of available Jenkins plugins by Jenkins developers and tested together for better quality.
+
+  <dt>Improved user interfaces
+  <dd>
+    A number of small but useful improvements are made to the user interfaces that
+    you touch everyday, such as the freestyle job configuration pages.
+
+  <dt>All the Jenkins goodness you've accustomed &amp; loves
+  <dd>
+    Jenkins 2.0 is a drop-in replacement of Jenkins 1.x and fully backward compatible.
```

Comment:
You're not going to lose access to your data by upgrading, and it doesn't change existing APIs around. Just new system requirements.